web/themes/custom/westy/src/scss/components/_back-to-top.scss

```diff
@@ -0,0 +1,6 @@
+#back-to-top {
+    position: fixed;
+    bottom: 4rem;
+    right: 9rem;
+    z-index: $zindex-popover;
+  }
```

Comment:
Needs a newline.

Reply:
Add newline to file

Reply:
It's odd that Github is still seeing that as not having a newline. 🤔

Reply:
@radmosley 's latest changes removed the newline warning for me.